unittest/FanOutDAQModule_test.cxx

```diff
@@ -51,8 +51,9 @@ BOOST_AUTO_TEST_CASE(Configure)
 {
   dunedaq::appfwk::FanOutDAQModule<dunedaq::appfwk::NonCopyableType> foum("test");
 
-  auto config = R"({"input": "input"})"_json;
-  foum.configure(config);
+  auto config = R"({"input": "input", "fanout_mode": "RoundRobin", "outputs": []})"_json;
+  foum.set_config(config);
```

Comment:
we should have here the same structure we have in the DAQApplication. simply call `do_init`.